### PR TITLE
Fix/time format

### PR DESCRIPTION
**Pull request by jamesjarvis**
fixes if it says for example:
- '1 songs'
- '0 hours 1 minutes'

### CODE REVIEW

File: src/app/utils/Utils.js [hunk 1]
```diff
@@ -21,13 +21,22 @@ export function getTime(ms) {
...
-  const minutes = parseInt((duration / (1000 * 60)) % 60, 10);
-  const hours = parseInt((duration / (1000 * 60 * 60)) % 24, 10);
+  const musickitDuration = MusicKit.formattedMilliseconds(duration);
 
-  let humanReadable = `${minutes} minute${minutes === 1 ? 's' : ''}`;
-  if (hours > 0) {
-    humanReadable = `${hours} hour${hours === 1 ? 's' : ''}, ${humanReadable}`;
-  }
+  const hourFormatted = musickitDuration.hours === 1 ? 'hour' : 'hours';
+  const hours = musickitDuration.hours === 0 ? `` : `${musickitDuration.hours} ${hourFormatted}`;
```
Comment (BrychanOdlum) [inline src/app/utils/Utils.js]: '' instead of `` for empty strings pls!

File: src/app/utils/Utils.js [hunk 2]
```diff
@@ -21,13 +21,22 @@ export function getTime(ms) {
...
+  const minuteFormatted = musickitDuration.minutes === 1 ? 'minute' : 'minutes';
+  const minutes =
+    musickitDuration.minutes === 0 ? `` : `${musickitDuration.minutes} ${minuteFormatted}`;
+  const comma = musickitDuration.hours === 0 || musickitDuration.minutes === 0 ? `` : `, `;
+  const humanReadable = `${hours}${comma}${minutes}`;
```
Comment (BrychanOdlum) [inline src/app/utils/Utils.js]: just return here, no need to declare another const

File: src/app/utils/Utils.js (hunk 1 above)
Comment (jamesjarvis) [reply src/app/utils/Utils.js]: oki

File: src/app/utils/Utils.js (hunk 2 above)
Comment (jamesjarvis) [reply src/app/utils/Utils.js]: tru tru

**Pull request merged**